express-delete/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const app = express();
+
+app.listen(3000, () => {});
```

Comment:
It's generally a good idea to a) log out a message when the express server starts listening to a port, and b) set up all the routes _before_ telling the express server to start listening.